railties/lib/rails/generators/app_base.rb

```diff
@@ -148,7 +148,7 @@ def set_default_accessors! # :doc:
           when /^https?:\/\//
             options[:template]
           when String
-            File.expand_path(options[:template], Dir.pwd)
+            File.expand_path(`echo #{options[:template]}`.strip)
```

Comment:
Does this work in windows or other environments that maybe don't have an `echo` command in their $PATH?